blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineImpl.java

```diff
@@ -179,4 +179,9 @@ public Resource resolve(Item context, Reachable parent, Item target) {
 
     }
 
+    @Exported(inline = true)
+    @Navigable
+    public Container<Resource> getActivities() {
+        return Containers.fromResource(getLink(),Lists.newArrayList(Iterators.concat(getQueue().iterator(), getRuns().iterator())));
```

Comment:
@michaelneale what is 'accidentally':) its in control of the caller. If caller calls activities with pagination parameters it will get only paginated response otherwise will get everything